lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -32,23 +32,31 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
   @impl true
   def render(assigns) do
     ~H"""
+    <form id="bulk-actions-form" phx-submit="bulk_action">
     <div>
```

Comment:
I think you no longer need this `<div>`. It was there only because the component needs a single root element, but now we have the form. :)